apps/extension/src/subndarray.cc

```diff
@@ -0,0 +1,25 @@
+#include <subndarray.h>
+#include <tvm/runtime/packed_func.h>
+#include <tvm/runtime/ndarray.h>
+#include <tvm/runtime/registry.h>
+
+namespace subndarray {
+
+using namespace tvm;
+using namespace tvm::runtime;
+
+NDSubClass::NDSubClass() : NDArray() {
+}
+
+NDSubClass::NDSubClass(NDArray::Container *data) : NDArray(data) {
+}
+
+NDSubClass::Container::Container() {
+}
+
+TVM_REGISTER_GLOBAL("subndarray.print_hello")
+.set_body([](TVMArgs args, TVMRetValue *rv) {
```

Comment:
add a test case where you return a SubClass and pass it back as args

Reply:
Will do after the python side is done